src/components/Avatar.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function Avatar({ image }) {
+  return <img className="ui mini avatar image" style={{ margin: 'auto' }} src={image} alt="" />;
```

Comment:
Do not use inline style. It makes your app slower because it has to be re-rendered. Move the margin:auto into a css class.